### DISCUSSION TITLE
zero123 checkpoints not found

### DISCUSSION THREAD

**Discussion by stvn037** [lllyasviel/stable-diffusion-webui-forge#507] [Q&A]
Hello here, just trying to get started testing it, using a runpod template.
They checkpoints are not showing in the dropdown, any tips on configuring?
<img width="287" alt="309442714-6d0ad1a4-9921-49a9-91c6-e47f66fc2b32" src="https://github.com/lllyasviel/stable-diffusion-webui-forge/assets/37275536/349c1748-5b58-4da0-a4ba-e077fd46cfce">
Under the z123 tab:
<img width="643" alt="309442700-df3f6085-65d8-42cd-8f03-f4ea3cf0ef8d" src="https://github.com/lllyasviel/stable-diffusion-webui-forge/assets/37275536/f1227a0b-e387-4766-9d51-98ce5a718d7f">

**Comment by leomaxwell973** [comment]
script for z123 - extensions-builtin\sd_forge_z123\scripts\forge_z123.py :

Lns 22-33
```
model_root = os.path.join(models_path, 'z123')
os.makedirs(model_root, exist_ok=True)
model_filenames = []


def update_model_filenames():
    global model_filenames
    model_filenames = [
        pathlib.Path(x).name for x in
        shared.walk_files(model_root, allowed_extensions=[".pt", ".ckpt", ".safetensors"])
    ]
    return model_filenames
```

LN 63
`                                           value=model_filenames[0] if len(model_filenames) > 0 else None)`
uh is that first 0 supposed to be there? I'm kind of a chump at python when it comes to the logic behind the syntax of the code.... but just don't seem right, from a chumps (thats me) perspective :P
